tests/functional/reporting-non-auth.js

```diff
@@ -190,6 +138,92 @@ define(
         );
       },
 
+      "Description validation": function() {
+        return (
+          FunctionalHelpers.openPage(
+            this,
+            url("/issues/new"),
+            ".wc-ReportForm-actions-button"
+          )
+            .findByCssSelector("#description")
+            .click()
+            .end()
+            .findByCssSelector("#steps_reproduce")
+            .click()
+            .end()
+            // .execute(function() {
+            //   var elm = document.querySelector("#steps_reproduce");
+            //   WindowHelpers.sendEvent(elm, "input");
+            // })
+            .sleep(800)
+            // using xpath because 2 errors display here
+            .findByCssSelector(
+              "div.is-error:nth-child(3) > div:nth-child(1) > span:nth-child(2)"
+            )
+            .getVisibleText()
+            .then(function(text) {
+              assert.include(
+                text,
+                "Description required.",
+                "Description validation message is shown"
+              );
+            })
+            .end()
+            .findByCssSelector("#description")
+            .type("bug description")
+            .end()
+            .waitForDeletedByCssSelector(
+              "div.is-error:nth-child(3) > div:nth-child(1) > span:nth-child(2)"
+            )
+            .end()
+        );
+      },
+
+      "(optional) browser + os validation": function() {
+        return (
+          FunctionalHelpers.openPage(
+            this,
+            url("/issues/new"),
+            ".wc-ReportForm-actions-button"
+          )
+            // make sure we can see the valid checkbox (i.e. it's background image is non-empty)
+            .execute(function() {
+              return window
+                .getComputedStyle(document.getElementById("span-os"), ":after")
+                .getPropertyValue("background-image");
+            })
+            .then(function(bgImage) {
+              assert.include(
+                bgImage,
+                "valid.svg",
+                "The valid checkbox psuedo is visible"
+              );
+            })
+            .end()
+            .execute(function() {
+              var elm = document.querySelector("#os");
+              elm.value = "";
+              WindowHelpers.sendEvent(elm, "input");
+            })
+            .end()
+            .sleep(500)
+            // make sure we can't see the valid checkbox (i.e. it's background image is empty)
+            .execute(function() {
+              return window
+                .getComputedStyle(document.getElementById("span-os"), ":after")
+                .getPropertyValue("background-image");
+            })
+            .then(function(bgImage) {
+              assert.notInclude(
+                bgImage,
+                "valid.svg",
+                "The valid checkbox psuedo is not visible"
```

Comment:
nit: s/psuedo/pseudo/

(I realize you inherited this typo, but might as well fix it while we're here. :))